kademlia/src/main/java/ingsw/group1/kademlia/KadActionsBuilder.java

```diff
@@ -28,7 +26,7 @@
      */
     public KadActionsBuilder(int maxActionID) {
         if (maxActionID >= KadAction.MIN_ID && maxActionID <= KadAction.MAX_ID)
-            MAX_ACTION_ID = maxActionID;
+            maxActionId = maxActionID;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4761148)